client/lib/components/home_page_sections/home_page_recent_numbers.dart

```diff
@@ -103,15 +103,14 @@ class __HomeStatsFaderState extends State<_HomeStatsFader>
         child: AnimatedCrossFade(
           firstChild: Observer(
             builder: (_) => _HomeStatCard(
-              stat: widget.statsStore.countryStats != null &&
-                      widget.statsStore.countryDailyCases > 0
+              stat: widget.statsStore.countryStats != null && widget.statsStore.countryDailyCases >= 0
```

Comment:
Revert styling change
```suggestion
              stat: widget.statsStore.countryStats != null &&
                      widget.statsStore.countryDailyCases > 0
```